h2o-core/src/main/java/hex/DMatrix.java

```diff
@@ -253,7 +253,6 @@ public VecTsk(H2OCountedCompleter cmp, Key progressKey, double [] y){
         final Chunk xChunk = chks[i];
         for (int k = xChunk.nextNZ(-1); k < res.length; k = xChunk.nextNZ(k))
           try { res[k] += yVal * xChunk.atd(k);} catch(Throwable t) {
```

Comment:
This doesn't really make much sense, I would just get rid off the whole try - catch